cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnTwillController.java

```diff
@@ -138,6 +139,14 @@ protected void doStartUp() {
         LOG.info("Yarn application {} {} is not in running state. Shutting down controller.",
                  appName, appId, Constants.APPLICATION_MAX_START_SECONDS);
         forceShutDown();
+      } else {
+        try {
```

Comment:
hmm, that is not part of this change. Perhaps a merge artifact. I will remove it.